xyz-hub-service/src/main/java/com/here/xyz/hub/config/dynamo/DynamoTagConfigClient.java

```diff
@@ -225,6 +225,17 @@ private static List<Tag> tagDataToTags(List<Map<String, AttributeValue>> items)
         .withId(tagData.get("id").getS())
         .withSpaceId(tagData.get("spaceId").getS())
         .withVersion(Long.parseLong(tagData.get("version").getN()))
-        .withSystem( tagData.get("system") != null ? tagData.get("system").getBOOL() : false )).collect(Collectors.toList());
+        .withSystem( tagData.get("system") != null ? tagData.get("system").getBOOL() : false )
+        .withDescription(tagData.get("description") != null ? tagData.get("description").getS() : "")
+        .withAuthor(tagData.get("author") != null ? tagData.get("author").getS() : "system")
+        .withCreatedAt(tagData.get("createdAt") != null ? Long.parseLong(tagData.get("createdAt").getN()) : Core.startOfYearMillis())
```

Comment:
Suggesting to rather return some "invalid" value (e.g. -1) to tell the user about the non-existence of the timestamp rather than responding with a wrong value.